torch/csrc/jit/passes/quantization.cpp

```diff
@@ -514,51 +511,48 @@ void InsertQuantDeQuantImpl(
     }
   }
 
-  std::vector<Value*> values_to_quantize;
-  std::unordered_map<script::ModulePtr, script::Module>
-      child_modules_to_quantize;
   QuantizeHelper qh(module);
   std::stack<Block*> blocks_to_visit;
   blocks_to_visit.push(graph->block());
   while (!blocks_to_visit.empty()) {
     Block* b = blocks_to_visit.top();
     blocks_to_visit.pop();
-    for (Node* n : b->nodes()) {
+    for (auto it = b->nodes().begin(), end = b->nodes().end(); it != end;) {
+      Node* n = *it++;
       for (Value* v : n->outputs()) {
         if (v->type()->isSubtypeOf(TensorType::get())) {
-          auto child_module = qh.findChildModuleToQuantize(v);
-          if (child_module) {
-            child_modules_to_quantize[child_module.value().module_object()] =
-                child_module.value();
+          if (v->node()->kind() == prim::CallMethod) {
```

Comment:
Nit: The if above can be changed to `if (!isTensor) { continue; ... }` so that we have less `}` braces in the code afterwards (it makes it easier to see what `if` an `else` relates to and in general makes the code more stream-lined).